example/docker-compose/readme.md

```diff
@@ -30,44 +30,46 @@ Tempo can be run with local storage, S3, Azure, or GCS backends. See below for e
 In this example all data is stored locally in the example-data/tempo folder.
 
 1. First start up the local stack. 
-```
+```console
 docker-compose up -d
 ```
 
 At this point, the following containers should be spun up -
 
 ```console
-$ docker-compose -f docker-compose.local.yaml ps
+docker-compose ps
+```
+```
                   Name                                 Command               State                         Ports
 --------------------------------------------------------------------------------------------------------------------------------------
 docker-compose_grafana_1                    /run.sh                          Up      0.0.0.0:3000->3000/tcp
```

Comment:
i've always found these sections that list every pod to be too much detail and cluttering the doc. what do you think?

should we remove?

Reply:
I personally don't mind it because it's a bit easier to follow along without executing all the commands. I more worried about the duplication on this page, every example follows the same format but has a slightly different command to check the storage.

Reply:
yeah, perhaps if we broke this into one .md for each example it would feel better?